src/main/java/seedu/progresschecker/model/ProgressChecker.java

```diff
@@ -219,23 +247,43 @@ public void closeIssueOnGithub(Index index) throws IOException, CommandException
      * Replaces the given person {@code target} in the list with {@code editedPerson}.
      * {@code ProgressChecker}'s tag list will be updated with the tags of {@code editedPerson}.
      *
-     * @throws DuplicatePersonException if updating the person's details causes the person to be equivalent to
-     *      another existing person in the list.
-     * @throws PersonNotFoundException if {@code target} could not be found in the list.
+     * @throws IOException if there is any problem in git authentication or parameter
      *
-     * @see #syncWithMasterTagList(Person)
      */
-    public void updatePerson(Person target, Person editedPerson)
-            throws DuplicatePersonException, PersonNotFoundException {
-        requireNonNull(editedPerson);
+    public void updateIssue(Index index, Issue editedIssue) throws IOException {
+        requireNonNull(editedIssue);
+        GitHub github = GitHub.connectUsingPassword(userLogin, userAuthentication);
+        GHRepository repository = github.getRepository(repoName);
+        GHIssue toEdit = repository.getIssue(index.getOneBased());
 
-        Person syncedEditedPerson = syncWithMasterTagList(editedPerson);
-        // TODO: the tags master list will be updated even though the below line fails.
-        // This can cause the tags master list to have additional tags that are not tagged to any person
-        // in the person list.
-        persons.setPerson(target, syncedEditedPerson);
+        List<Assignees> assigneesList = editedIssue.getAssignees();
+        List<Labels> labelsList = editedIssue.getLabelsList();
+
+        ArrayList<GHUser> listOfUsers = new ArrayList<>();
+        ArrayList<String> listOfLabels = new ArrayList<>();
+        MilestoneMap obj = new MilestoneMap();
+        HashMap<Milestone, Integer> getMilestone = obj.getMilestoneMap();
+
+        for (int ct = 0; ct < assigneesList.size(); ct++) {
+            listOfUsers.add(github.getUser(assigneesList.get(ct).toString()));
+        }
+
+        for (int ct = 0; ct < labelsList.size(); ct++) {
+            listOfLabels.add(labelsList.get(ct).toString());
+        }
+
+        if (editedIssue.getMilestone() != null) {
+            GHMilestone check = repository.getMilestone(getMilestone.get(editedIssue.getMilestone()));
+            toEdit.setMilestone(check);
+        }
+        toEdit.setTitle(editedIssue.getTitle().toString());
+        toEdit.setBody(editedIssue.getBody().toString());
+        toEdit.setAssignees(listOfUsers);
+        toEdit.setLabels(listOfLabels.toArray(new String[0]));
     }
 
+    //@@author
+
```

Comment:
this is for?

Reply:
To set a breakpoint up till where his code contribution is.

Reply:
yep